models/application.py

```diff
@@ -67,10 +69,10 @@ def remove(self, name):
     def restart(self, name):
         app = self.get(name)
```

Comment:
Removed `if`. No need for `except` block here, `get` itself raises a correct exception